atd-etl/cris_import/cris_import.py

```diff
@@ -107,9 +115,45 @@ def main():
             typed_token = align_db_typing(trimmed_token)
             align_records_token = align_records(typed_token)
             clean_up_import_schema(align_records_token)
+        mark_extract_as_imported(archive_data[0], database_location)
```

Comment:
Here we are using the ID of the record to form up the `update` statement that marks a successful import of the extract.